### PR TITLE
Added positions submitted via email

### PR DESCRIPTION
**Pull request by ReynaldoBoulogne**

### CODE REVIEW

File: _companies/cybercom.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: BYON8
```
Comment (Jim-Holmstroem) [inline _companies/cybercom.md]: cybercom?

**Comment by ReynaldoBoulogne** [comment]
yes! cybercom

*Reynaldo Boulogne*
E-Mail: reynaldo.boulogne
@gmail.com
Mob: +46 (0) 70 022 53 93 <+46%2070%20022%2053%2093>
LinkedIn: https://www.linkedin.com/in/reynaldoboulogne/
Website: https://reynaldoboulogne.wordpress.com/

**Comment by ReynaldoBoulogne** [comment]
The cybercome file is wrong. I will submit it again

**Comment by Jim-Holmstroem** [comment]
fixed by https://github.com/Stockholm-AI/stockholm-ai/pull/172

**Pull request closed without merging**